src/shared/components/dropdown/BasicDropdown.scss

```diff
@@ -1,20 +1,69 @@
+/* Basic Dropdown Styles */
 .basic-dropdown {
-  min-width: 100%;
-  button {
-    min-width: 100%;
-  }
+  width: 100%;
+  position: relative !important;
+}
 
-  &[aria-invalid=true] {
-    --pf-v5-c-dropdown__toggle--before--BorderBottomColor: var(--pf-v5-global--danger-color--100);
-    --pf-v5-c-dropdown__toggle--hover--before--BorderBottomColor: var(--pf-v5-global--danger-color--100);
-    --pf-v5-c-dropdown__toggle--focus--before--BorderBottomColor: var(--pf-v5-global--danger-color--100);
-    --pf-v5-c-dropdown__toggle--active--before--BorderBottomColor: var(--pf-v5-global--danger-color--100);
-    --pf-v5-c-dropdown--m-expanded__toggle--before--BorderBottomColor: var(--pf-v5-global--danger-color--100);
-
-    button {
-      &::before {
-        border-bottom-width: var(--pf-v5-global--BorderWidth--md);
-      }
-    }
+/* Dropdown Toggle Styles */
+.dropdown-toggle {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md);
+  border: var(--pf-v5-global--BorderWidth--sm) solid var(--pf-v5-global--BorderColor--300);
+  border-bottom-color: var(--pf-v5-global--BorderColor--200);
+  background-color: var(--pf-v5-global--Color--white);
+  cursor: pointer;
+  line-height: var(--pf-v5-global--LineHeight--sm); // var(--pf-v5-c-dropdown__toggle--LineHeight) is higher.
+  height: var(--pf-v5-global--LineHeight--sm);
+
+  /* Arrow Styles */
+  &::after {
+    content: '▼';
+    font-size: var(--pf-v5-global--FontSize--xs);
+    line-height: var(--pf-v5-global--LineHeight--sm);
+    margin-left: var(--pf-global--spacer--sm);
+    transform: scale(1.2, 0.5);
```

Comment:
Custom CSS shouldn't be needed for base components. Please use Patternfly components.
this will create problems in future when upgrading to a latest version of patternfly

Reply:
have migrated to select component. With the select component, we do not need to specific these kinds of format. 
Thank you for pointing out the direction.